src/js/components/Select/stories/FooterOption.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { Box, Text, Grommet, Select } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+const Option = React.memo(({ label, isFooter }) => (
+  <>
+    {isFooter ? (
+      <Box border="top">
```

Comment:
Can we do anything to cause the text to wrap as opposed to increasing be drop width? The single line message looks a bit long to me. Perhaps use `<Paragraph />`? Just wondering.